package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "safe-react",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
I think it should be, 2.2.0, we are adding new "features"

Reply:
It's done